image/verify/src/verifier.rs

```diff
@@ -714,7 +714,8 @@ impl<Env: ImageVerificationEnv> ImageVerifier<Env> {
                 Err(CaliptraError::IMAGE_VERIFIER_ERR_FMC_SVN_GREATER_THAN_MAX_SUPPORTED)?;
             }
 
-            if verify_info.svn < verify_info.min_svn {
+            // TODO: remove this check.
+            if verify_info.svn < verify_info.deprecated_min_svn {
```

Comment:
Aha, so we are looking at this field...

Reply:
Yes but it's just a sanity check, comparing two numbers in the signed header against each other. No other logic being performed on the field. Would be a safe two-liner change to just remove this field.

Reply:
Can we remove this?

Reply:
Will do in a follow-up.